src/components/label/index.tsx

```diff
@@ -7,7 +7,7 @@ export interface LabelProps extends Omit<FlexProps, 'css' | 'action'> {
   text?: string;
   pointer?: boolean;
   align?: 'top' | 'left';
-  children: React.ReactNode;
```

Comment:
it's possible to pass children-ish thing as "text", so `children` is not a required prop